examples/COLAB/COLAB_DEMO_mouse_openfield.ipynb

```diff
@@ -158,21 +131,21 @@
     },
     {
       "cell_type": "code",
+      "execution_count": null,
       "metadata": {
         "id": "_pOvDq_2oEJW"
       },
+      "outputs": [],
       "source": [
         "#let's also change the display and save_iters just in case Colab takes away the GPU... \n",
         "#if that happens, you can reload from a saved point. Typically, you want to train to 200,000 + iterations.\n",
-        "#more info and there are more things you can set: https://github.com/AlexEMG/DeepLabCut/blob/master/docs/functionDetails.md#g-train-the-network\n",
+        "#more info and there are more things you can set: https://github.com/DeepLabCut/DeepLabCut/wiki/DOCSTRINGS#train_network\n",
```

Comment:
fixed this broken link - all other changes in this notebook were because of VS Code